localisation/locale-deDE.lua

```diff
@@ -0,0 +1,225 @@
+--[[
+    localisation\template.lua
+
+    The template that all other localisation files hould be based off of.
+]]
+local _, Loc = ...;
+
+--[[
+    'frFR' French (France)
+    'deDE' German (Germany)
+    'enGB  English (Great Brittan) if returned, can substitute 'enUS' for consistancy
+    'enUS' English (America)
+    'itIT' Italian (Italy)
+    'koKR' Korean (Korea) RTL - right-to-left
+    'zhCN' Chinese (China) (simplified) implemented LTR left-to-right in WoW
+    'zhTW' Chinese (Taiwan) (traditional) implemented LTR left-to-right in WoW
+    'ruRU' Russian (Russia)
+    'esES' Spanish (Spain)
+    'esMX' Spanish (Mexico)
+    'ptBR' Portuguese (Brazil)
+]]
+if (GetLocale() == 'deDE') then
```

Comment:
Correct me if I am wrong, but I think here should be comparison with `frFR`. Filename should be corrected to `locale-frFR.lua` also, as you translating to French.